extensions/sql-migration/src/constants/strings.ts

```diff
@@ -31,6 +31,10 @@ export const SKU_RECOMMENDATION_ASSESSMENT_ERROR = (serverName: string): string
 	return localize('sql.migration.wizard.sku.assessment.error', "An error occurred while assessing the server '{0}'.", serverName);
 };
 export const SKU_RECOMMENDATION_CHOOSE_A_TARGET = localize('sql.migration.wizard.sku.choose_a_target', "Choose your Azure SQL target");
+export const SKU_RECOMMENDATION_SUBSCRIPTION_INFO = localize('sql.migration.sku.subscription', "Subscription");
```

Comment:
These are temporary text strings for info tooltips - the same as the current label value. I am going to create another PR with more specific tooltips once those are provided. So, I think specific names are required here.

Reply:
It doesn't really matter whether they're "temporary" or not. It's more that there's no reason to tie a string to a specific place that string is used. It just makes it more annoying later on if you want to use that same string elsewhere (in addition to the other issues I detailed above)

Reply:
I think it depends on the values we'll put instead of "Subscription" and "Choose your target" in the next PR. If those qualify for common strings, I'll update the names.
What do you think @brian-harris ?

Reply:
let's see what feedback we get from the content request from PO.